common/src/main/java/com/kin/ecosystem/common/model/NativeOffer.java

```diff
@@ -2,108 +2,111 @@
 
 public class NativeOffer {
 
-    private String id;
-    private String title;
-    private String description;
-    private int amount;
-    private String image;
-    private final OfferType offerType;
-
-    public NativeOffer(String id, OfferType offerType) {
```

Comment:
why did you remove the offer type? it's it necessary for the offer, it can cause a crash.